backend/src/routes/BankingRoutes.ts

```diff
@@ -61,7 +62,8 @@ async function get(_: IReq, res: IRes) {
 
     const createTokenResponse = await client.linkTokenCreate(configs);
     const data = createTokenResponse.data as BankingRoutesLinkData;
-    res.json(data.linkToken);
+    console.log(data);
```

Comment:
Nit: consider using a logging library instead of just console.log

Reply:
Will do, but decided to store the data in a separate variable for now